pkg/api/types/container_types.go

```diff
@@ -264,6 +264,9 @@ type ContainerCreateOptions struct {
 
 	// ImagePullOpt specifies image pull options which holds the ImageVerifyOptions for verifying the image.
 	ImagePullOpt ImagePullOptions
+
+	// Userns name for user namespace mapping of container
+	Userns string
```

Comment:
```suggestion
	UserNS string
```

Because `Userns` looks like `Users`

Reply:
Ack